extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -417,6 +423,12 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, env => {
         );
         return check('$MATCH(thisisatest, thisisatest, test)', 'thisisatest');
       });
+
+      it('"COOKIE" resolves cookie value', async () => {
```

Comment:
Since you have this, is it possible to 1. fake the element to be in FIE, 2. fake the runtime mode to be inabox, 3. fake the window.location to be a proxy url
in order to test the COOKIE feature?

Reply:
Happy to write some more tests. I think this seems more like testing the `cookieReader` function, rather than checking variable replacement. what do you think about doing this in `test-cookies.js`?

Reply:
or maybe `test-cookie-reader`?

Reply:
Talked offline, added to `test-variables.js` Done. @zhouyx please make sure I covered everything you are asking for.